src/main/flight/pid.c

```diff
@@ -742,25 +743,64 @@ static FAST_RAM_ZERO_INIT float setPointDAttenuation[XYZ_AXIS_COUNT];
 static FAST_RAM_ZERO_INIT timeUs_t crashDetectedAtUs;
 static FAST_RAM_ZERO_INIT timeUs_t previousTimeUs;
 
-    void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *angleTrim, timeUs_t currentTimeUs)
+static void processIterm(
+    uint8_t axis,
+    float previousIterm, 
```

Comment:
Would you think of changing the name of this? Since iterm is cumulative seeing accumulatedIterm, or some other term would feel more intuitive. When I first read the code I was thinking previous iterm was the previous change to iterm rather than the old value for iterm.

Reply:
why not some people say that the hardest part of programming in how to name things

Reply:
how about currentIntegral?

Reply:
I like it.